client/verta/tests/unit_tests/test_deployed_model_class.py

```diff
@@ -0,0 +1,250 @@
+# -*- coding: utf-8 -*-
+""" Unit tests for the verta.deployment.DeployedModel class. """
+
+import os
+import pytest
+import responses
+
+from responses import matchers
+from requests import Session
+from requests.exceptions import RetryError, HTTPError
+from typing import Any, Dict, List
+from unittest import TestCase
+from unittest.mock import Mock, patch
+from urllib3 import Retry
+
+from verta.credentials import EmailCredentials
+from verta.deployment import DeployedModel
+from verta._internal_utils import http_session_utils
+
+PREDICTION_URL: str = 'https://test.dev.verta.ai/api/v1/predict/test_path'
+TOKEN: str = '12345678-xxxx-1a2b-3c4d-e5f6g7h8'
+MOCK_RETRY: Retry = http_session_utils.retry_config(max_retries=http_session_utils.DEFAULT_MAX_RETRIES,
+                                            status_forcelist=http_session_utils.DEFAULT_STATUS_FORCELIST,
+                                            backoff_factor=http_session_utils.DEFAULT_BACKOFF_FACTOR
+                                            )
+MOCK_SESSION: Session = http_session_utils.init_session(retry=MOCK_RETRY)
+VERTA_CLASS = 'verta.deployment._deployedmodel'
```

Comment:
fyi, this is inconsistent with how we defined this variable [in the endpoint unit tests](https://github.com/VertaAI/modeldb/blob/121254d9de29ebfba673ed6d233b7581a1da8d38/client/verta/tests/unit_tests/test_endpoint_class.py#L36), where `VERTA_CLASS` includes the trailing comma:

```python
VERTA_CLASS: str= 'verta.endpoint.Endpoint.'
```

(also `_deployedmodel` is a module rather than a class 😆)

Reply:
Good points.  I'll address both